tests/foreman/api/test_contentview.py

```diff
@@ -250,6 +250,30 @@ def setUpClass(cls):  # noqa
         with open(get_data_file(PUPPET_MODULE_NTP_PUPPETLABS), 'rb') as handle:
             cls.puppet_repo.upload_content(handle)
 
+    def add_content_views_to_composite(self, composite_cv, cv_amount=1):
+        """
+        Add necessary number of content views to the composite one
+
+        :param composite_cv: Composite content view object
+        :param cv_amount: Amount of content views to be added
+        """
+        cvv_ids = []
+        for _ in range(cv_amount):
+            content_view = entities.ContentView(
+                organization=self.org.id
+            ).create()
+            content_view.publish()
+            cvv_ids.append(content_view.read_json()['versions'][0]['id'])
+        client.put(
+            composite_cv.path(),
+            {'content_view': {'component_ids': cvv_ids}},
```

Comment:
When updating a content view, there's no need to use a nested hash. This can be re-written like so:

    {'component_ids': cvv_ids}

I'm basing the above statement on the contents of the API docs.